### PR TITLE
replace particle effects with animated sprite for rocket explosion.

### PR DESCRIPTION
**Pull request by allisonchristensen**
deleted the old particle effects scenes as we won't be needing these. Also added sfx for launch and explosion of rocket.

### CODE REVIEW

**Comment by fheysel** [comment]
Code looks good to me, I'd just check with @charasyn that its visible on the pi

**Comment by charasyn** [comment]
I just tried this and I actually can't run it. I think the sound effect file is missing.
```
ERROR: Failed loading resource: res://sfx/rocket_launch_sfx.wav. Make sure resources have been imported by opening the project in the editor at least once.
   At: core/io/resource_loader.cpp:278
ERROR: poll: res://RocketEnemy.tscn:5 - Parse Error: [ext_resource] referenced nonexistent resource at: res://sfx/rocket_launch_sfx.wav
   At: scene/resources/resource_format_text.cpp:440
```

**Comment by charasyn** [comment]
By removing the references to the sound effects (so there are just no sound effects) I have it working on my local machine. About to test on Pi.
EDIT: After testing on the Pi, it seems like there are multiple rocket textures that are not showing up, including the texture of the rocket itself.
EDIT, again: Ok, upon looking closer all of the animated effects are showing properly. It's just the rocket that's not showing. This PR should be good to add after adding the sound effects and rebasing.

File: godotproject/sfx/explosion_sfx.wav.import [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[remap]
```
Comment (charasyn) [inline godotproject/sfx/explosion_sfx.wav.import:1]: The .wav files themselves don't seem to exist in the branch. Can you add them in?

**Review by charasyn — CHANGES_REQUESTED** [review]
Should be good to merge after adding the audio files.

File: godotproject/sfx/explosion_sfx.wav.import (hunk 1 above)
Comment (allisonchristensen) [reply godotproject/sfx/explosion_sfx.wav.import:1]: yep, will do!

**Review by fheysel — APPROVED** [review]

**Pull request merged**